tfs/reader.py

```diff
@@ -33,43 +50,45 @@ def read_tfs(
     tfs_file_path: pathlib.Path | str,
     index: str | None = None,
     non_unique_behavior: str = "warn",
-    validate: bool = True,  # noqa: FBT001, FBT002
+    validate: str | None = None,
```

Comment:
**NOTE**: We now skip validation by default when reading, as we assume the user trusts their TFS file is correct. This can always be turned on, but saves a lot of time on very big dataframes (such as sliced FCC lattices etc).